lib/code.rb

```diff
@@ -0,0 +1,53 @@
+require_relative './commands'
+
+class Code
+  include Commands
+  attr_reader :red_pins, :white_pins
+
+  def generate_code
+    @colors = colors = ["r", "g", "o", "y", "b", "p"]
+    @answer = (1..4).map { @colors[rand(6)] }
```

Comment:
Can we make these numbers more meaningful/explicit? Can we use constants if these don't really change?